core/templates/pages/delete-account-page/delete-account-page.component.html

```diff
@@ -49,4 +49,12 @@ <h2 translate="I18N_DELETE_ACCOUNT_PAGE_PARAGRAPH_1_HEADING"></h2>
     justify-content: center;
     margin: 30px 0;
   }
+
+  .oppia-delete-account ul {
+    margin-left: 40px;
+  }
+
+  .oppia-delete-account list {
```

Comment:
This is wrong. You are missing a "." before the class name.

Reply:
@Hudda ping.